.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: java
-dist: trusty
+dist: xenial
```

Comment:
I always get confused by these short names and get to google them :-) 

It's 16.04, and I guess it's become available quite recently.

Reply:
Yes, but not on Travis